<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lingue_table</class>
 <widget class="QWidget" name="lingue_table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>871</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_cancel_lingue_table">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_aggiorna_lingue_table">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Aggiorna Lingua</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_aggiungi_lingue_table">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>270</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Aggiungi Lingua</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
